### PR TITLE
Update ancestor and children widget endpoints

### PR DESCRIPTION
**Pull request by jhrr**
Changes necessary to synchronise with upstream ATLAS [`Node` work](https://github.com/scaife-viewer/explorehomer-atlas/pull/16).

See also: https://github.com/scaife-viewer/explorehomer/pull/2

### CODE REVIEW

File: src/widgets/PassageAncestorsWidget.vue [hunk 1]
```diff
@@ -33,7 +33,7 @@
         return this.passage
           ? gql`
             {
-              passageLines(reference: "${this.passage.absolute}") {
+              passageTextParts(reference: "${this.passage.absolute}") {
```
Comment (jacobwegner) [inline src/widgets/PassageAncestorsWidget.vue]: `ancestorsLens` still references `passageLines` below

File: src/widgets/PassageChildrenWidget.vue [hunk 2]
```diff
@@ -32,7 +32,7 @@
         return this.passage
           ? gql`
             {
-              passageLines(reference: "${this.passage.absolute}") {
+              passageTextParts(reference: "${this.passage.absolute}") {
```
Comment (jacobwegner) [inline src/widgets/PassageChildrenWidget.vue]: `childrenLens` still references `passageLines` below

File: src/widgets/PassageAncestorsWidget.vue (hunk 1 above)
Comment (jhrr) [reply src/widgets/PassageAncestorsWidget.vue]: achh you are right, I was sure I had also updated that... fixing

**Comment by jhrr** [comment]
> @jhrr see my comments;
> 
> Also, how have you been debugging `scaife-widgets` locally? I was having a terrible time getting things to work with `yarn link` (hit some kind of recursion issue), so I ended manually editing files in `node_modules` and copying the output of `npm build` from `scaife-widgets`.

Yes, @mjhea0 and I were doing the same sort of thing when we were working across the split, we didn't come up with anything better!

**Comment by jhrr** [comment]
> > @jhrr see my comments;
> > Also, how have you been debugging `scaife-widgets` locally? I was having a terrible time getting things to work with `yarn link` (hit some kind of recursion issue), so I ended manually editing files in `node_modules` and copying the output of `npm build` from `scaife-widgets`.
> 
> Yes, @mjhea0 and I were doing the same sort of thing when we were working across the split, we didn't come up with anything better!

I am going to try out `yarn link` now myself.

**Comment by jacobwegner** [comment]
LGTM @jhrr; let's merge and move forward with a new NPM release

**Pull request merged**